src/test/java/org/gridsuite/study/server/StudyServiceTests.java

```diff
@@ -0,0 +1,175 @@
+/*
+  Copyright (c) 2022, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server;
+
+import com.powsybl.iidm.network.VariantManagerConstants;
+import nl.jqno.equalsverifier.EqualsVerifier;
+import org.gridsuite.study.server.dto.EquipmentInfos;
+import org.gridsuite.study.server.dto.TombstonedEquipmentInfos;
+import org.gridsuite.study.server.dto.VoltageLevelInfos;
+import org.gridsuite.study.server.elasticsearch.EquipmentInfosService;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.elasticsearch.NoSuchIndexException;
+import org.springframework.test.context.junit4.SpringRunner;
+import reactor.core.publisher.Mono;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Nicolas Noir <nicolas.noir at rte-france.com>
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE, properties = {"spring.data.elasticsearch.enabled=true"})
+public class StudyServiceTests {
```

Comment:
Maybe rename into StudySearchServiceTests ?

Reply:
StudyServiceSearchTests would be ok ? (searching using StudyService)

Reply:
Done